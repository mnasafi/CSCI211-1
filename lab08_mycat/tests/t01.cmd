tests/t01.in results/t01.myout
