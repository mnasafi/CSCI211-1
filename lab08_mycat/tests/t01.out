Dennis Richie Quotes:

"UNIX is very simple, it just needs a genius to understand its simplicity."  

"C is quirky, flawed, and an enormous success."

C has "the power of assembly language and the convenience of... assembly language."
